tests/Unit/Models/MarketModelTest.php

```diff
@@ -25,6 +25,8 @@ public function testMarketIsCreatedWithExpectedAttributes()
         $this->assertInstanceOf(Market::class, $m);
         $this->assertNotNull($m->name);
         $this->assertNotNull($m->location);
+        $this->assertNotNull($m->payment_message);
+        $this->assertNotNull($m->sponsor_shortcode);
```

Comment:
You have also added 2 methods to the Market model. Write basic tests for them. (See Trader and Voucher model tests for example)